app/app.go

```diff
@@ -1220,8 +1228,11 @@ func (app *Evmos) setupUpgradeHandlers() {
 		// no store upgrade in v9 or v9.1
 	case v10.UpgradeName:
 		// no store upgrades in v10
+	case v11.UpgradeNameRC3:
+		// rename recovery store
+		// TODO: Not sure what goes here
```

Comment:
You'll have to replace `storetypes.StoreUpgrades` by the one defined on `app/store_loader.go`
```suggestion
storeUpgrades = &storetypes.StoreUpgrades{
			Replaced: []storetypes.StoreRename{
				{
					OldKey: "recovery",
					NewKey: "recovery",
				},
			},
		}
```